### ISSUE TITLE
Investigate why there's missing alt text in Featured Image

### ISSUE THREAD

**Issue by brandonhundt** [PRX/theworld.org#198]
1. go here: https://preview.theworld.org/stories/2021-07-27/climate-change-driving-worst-drought-madagascar-has-seen-four-decades
2. Featured image is missing alt text

(Noted in multiple stories)
